uwblueprint/headsupguys
# Fix/add updated at to tool schema and refactor seeding script

## Notion ticket link
[Ticket Name](https://uwblueprintexecs.notion.site/Refactor-seeding-script-to-use-Mongoose-7b6c62965283461bb7fe58a1d40bf883)

LINKED TO BUG: https://uwblueprintexecs.notion.site/Add-updatedAt-field-to-Tool-Schema-aa56d53fcec8466e8c269246d06027b3


## Implementation description
* Refactored MongoDB seeding script to implement Mongoose, allowing for updatedAt parameter to successfully appear.
* Implemented new updatedAt parameter in tools


## Steps to test
1. Run `yarn seed` in terminal to see if MongoDB successfully seeds with updatedAt parameter
2. Check endpoint `admin/dashboard/tools` to see if creating a new tool gives a correct date parameter at bottom right


## What should reviewers focus on?
* Ensuring seeding script still works as intended for both `yarn seed` and `yarn seed --init`


## Checklist
